### PR TITLE
Improving Buying Guide user flow and layout

### PR DESCRIPTION
**Pull request by rickychilcott**
@benlachman this is taking shape.

- [ ] Get Nate to remove background tan color from meter types
- [ ] make meter icons the same size
- [ ] remove the detailed text in step two and replace with the image of the CC
- [x] make reset -> "start over" and a text link instead of button.
- [x] change step three to ask as a positive question (e.g. Is the space within 4ft clear…) so that yeses result in positive outcome.
- [x] make the go buy glow into a nice big button. And the congratulatory text larger
- [x] test on mobile (wasn't working yesterday)
- [x] disable links after a user selects a response for each question

### CODE REVIEW

**Comment by benlachman** [comment]
- Get Nate to remove background tan color from meter types
- make reset -> "start over" and a text link instead of button. 
- make meter icons the same size
- test on mobile (wasn't working yesterday)
- remove the detailed text in step two and replace with the image of the CC
- change step three to ask as a positive question (e.g. Is the space within 4ft clear…) so that yeses result in positive outcome. 
- make the go buy glow into a nice big button. And the congratulatory  text larger.

**Comment by rickychilcott** [comment]
OK, I made several changes to the UI, support mobile now, and other minor changes.

I think designing of graphics is all that really remains.

@benlachman are you going to reach out to Nate or shall I? The graphics should be built as the same size, as to fix the sizing problems you experienced before. Also, I haven't seen a credit card graphic anywhere, but maybe I've missed it. Not sure if we have something already.

One other thought/question. When people get to question-3, should we be logging it anywhere? I'm wondering if we want to capture which paths people go down to understand where to focus our attention next?

File: buying-guide.html [hunk 1]
```diff
@@ -8,78 +8,129 @@
...
+  			<h2>Glow is compatible!</a></h2>
+  		</header>
 
-		<p>
-			We're sorry, but your home is not compatible. Please signup below for details about when we expand abilities to support your house.
-		</p>
-	</section>
-</div>
+  		<p>
+  			Yay! Glow is compatabile with Glow. You'll be saving money and helping the environment before you know it. <br /><a class="button special fit cta" href="{{ site.share.kickstarter }}">Buy Now</a>
```
Comment (cantino) [inline buying-guide.html]: "Glow is compatible with Glow"?

Comment (rickychilcott) [reply buying-guide.html]: Ha! Good catch. I started with the idea of Home being compatible with a Glow, but then switched it and must have missed it. Clearly, what I wrote doesn't make sense, but how should we think about it?

**Comment by rickychilcott** [comment]
@benlachman have you had a chance to review this and figure out how/who/etc of graphics? I'm just not sure who has done what design-wise.

File: _data/buying_guide.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@
       result: question-failure
 - id: 2
   question:
-    helper_text: We need at least enough space for a credit card to lie flat on the meter panel surface above the meter without interference from a) the contoured circular flange around the meter and b)flange extending from the top of the meter panel
+    helper_text: We need at least enough space for a credit card to lie flat aginst the meter panel surface without interference.
```
Comment (cantino) [inline _data/buying_guide.yml]: Typo in `aginst`

**Pull request merged**